UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/Admin/RadioNavigationalWarningAdminTest.cs

```diff
@@ -0,0 +1,158 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning;
+using UKHO.MaritimeSafetyInformation.Web.Controllers;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.RadioNavigationalWarning.Admin
+{
+    [TestFixture]
+    public class RadioNavigationalWarningAdminTest : BaseRNWTest
```

Comment:
Can you please add a test that checks that the complete list of RNWs is returned when the list is split into pages? e.g. if "records per page" = 4, and you have 10 records, can you check that get all these 10 records appear on the 3 pages?  This is to avoid indexing errors.

Reply:
Thanks for addressing.
 
We have handle the above case under **WhenCallIndexWithValidPageNo_ThenReturnAsyncListBasedOnPageNo** test case.

From server-side we are only fetching records based on page no, that's what we check the total page count in this test. And also we have checked the record on page no 2.

Thanks